src/style.css

```diff
@@ -1 +1,70 @@
 /* Styles go here */
+
+* {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
That's a bad idea to reset paddings and margins for every element
Moreover, you don't need to reset paddings
On the other hand, this selector is for setting `box-sizing`

Fix it by moving it under a different selector